sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogOptions.java

```diff
@@ -13,9 +15,13 @@
  * The log configurations for HTTP messages.
  */
 public class HttpLogOptions {
+    private String applicationId;
```

Comment:
Should we rename this file to `HttpClientOptions`?

Reply:
That would be an API breaking change